### PR TITLE
Removing the need to maintain lodash fork

### PR DESCRIPTION
**Pull request by jasonmit**
This just removes the need for you to maintain the fork and makes lodashes the "source of truth".

Great addon :+1:

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+var mergeTrees = require('broccoli-merge-trees');
+
+var find = stew.find;
+
 module.exports = {
-  name: 'lodash'
+  name: 'lodash',
+
+  treeForAddon: function(tree) {
+    var lodashPath = path.join(this.project.nodeModulesPath, 'lodash-es');
```
Comment (jasonmit) [inline index.js]: May be incorrect, let me validate.

File: index.js [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
 
+var path = require('path');
+var mergeTrees = require('broccoli-merge-trees');
+
 module.exports = {
-  name: 'lodash'
+  name: 'lodash',
+
+  treeForAddon: function(tree) {
+    var lodashPath = path.join(this.project.root, 'node_modules', 'ember-lodash', 'node_modules', 'lodash-es');
```
Comment (jasonmit) [inline index.js]: We can't `require.resolve` here unfortunately.  This is the best I got for determining the path.  Should be fine though.

File: addon/internal/root.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+/** Used to determine if values are of the language type `Object`. */
```
Comment (jasonmit) [inline addon/internal/root.js]: Pulled from your repo, nothing was touched here.

File: index.js [hunk 4]
```diff
@@ -1,6 +1,25 @@
...
+
+    if (this.app.name === 'dummy') {
+      lodashPath = path.join(this.project.root, 'node_modules', 'lodash-es');
+    }
+
+    var lodashTree = this.treeGenerator(lodashPath);
+
+    var trees = mergeTrees([lodashTree, tree], {
```
Comment (jasonmit) [inline index.js]: addon tree wins out over the lodash-es tree, so you're able to overwrite the bits that do not work, like `addon/internal/root.js`

**Comment by truenorth** [comment]
I think we've got a better solution on the way now that @jdalton has [made a fix to esperanto](https://github.com/esperantojs/esperanto/issues/151) and [I've released a new broccoli/es6-modules](https://github.com/ember-cli/broccoli-es6modules/pull/23).

**Comment by truenorth** [comment]
@jasonmit looks like we're a little ways out to the proper solution. This is still a significant improvement. Thanks!

**Comment by jasonmit** [comment]
https://github.com/truenorth/ember-lodash/pull/3 addresses one thing I noticed around how I build up the path

**Pull request merged**